### PR TITLE
Improve Icon story design 

### PR DESCRIPTION
**Pull request by KarnellSchultz**
## Describe your changes

- Changed the design of the icon page.
- Added the ability to click on the icon and have it's name copied to your clipboard.

Stole a bunch of ideas and code from @gustaveen 's Color story to make this happen. 

_“good artists borrow, great artists steal.”_ - [Pablo Picasso](https://media4.giphy.com/media/l2JhyuVn311Y1RWqA/200.gif) (maybe?? maybe not?? )

Screenshot 2023-01-04 at 12 06 46

_Would be cool to have a little toast show up when you've copied something to your clipboard but I did not want to add it to the project._

## Justify why they are needed

Not really needed. Just nice to have.

## Checklist before requesting a review

- [x] I have performed a self-review of my code

### CODE REVIEW

File: packages/ui/src/icons/Icons.stories.tsx [hunk 1]
```diff
@@ -12,30 +12,74 @@ const Icon = ({ icon }: IconProps) => {
...
-          <Icon icon={icon} />
-        </IconWrapper>
-      ))}
-    </>
+const IconName = styled.div({
+  paddingTop: '1rem',
+})
+
+const updateClipboard = (newClip: string) => {
+  navigator.clipboard.writeText(newClip).then(
```
Comment (robinandeer) [inline packages/ui/src/icons/Icons.stories.tsx:77]: nit: why not async await?

**Review by robinandeer — APPROVED** [review]
Great stuff!

[fuego](https://app.graphite.dev/macros/HedvigInsurance)

File: packages/ui/src/icons/Icons.stories.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply packages/ui/src/icons/Icons.stories.tsx]: copy/pasta 🍝 from mdn

**Review by guilhermespopolin — APPROVED** [review]
🍝

**Pull request merged**